### PR TITLE
OTC-800: modular IMIS: renewing of free enrolment policies doesn't work

### PR DESCRIPTION
**Pull request by hirensoni913**
https://openimis.atlassian.net/browse/OTC-800

Fixed the issue when selecting an officer start date of the policy becomes null.

### CODE REVIEW

**Comment by delcroip** [comment]
@dborowiecki @hirensoni913  can you check why the code you commented was put there in the first place ?

File: src/components/PolicyForm.js [hunk 1]
```diff
@@ -130,13 +130,14 @@ class PolicyForm extends Component {
...
           newPolicy: !this.props.renew,
           renew: false,
-        },
-        (e) => {
-          if (policy.stage === POLICY_STAGE_RENEW) {
-            policy.startDate = policy.expiryDate;
-            this.props.fetchPolicyValues(policy);
-          }
         }
+        // ,
```
Comment (dragos-dobre) [inline src/components/PolicyForm.js:134]: Remove commented code.

**Review by dragos-dobre — CHANGES_REQUESTED** [review]

**Comment by hirensoni913** [comment]
> 

@dborowiecki This block of code was duplicated

File: src/components/PolicyForm.js (hunk 1 above)
Comment (hirensoni913) [reply src/components/PolicyForm.js]: Done

**Review by delcroip — APPROVED** [review]

**Pull request merged**